example.scrutiny.yaml

```diff
@@ -26,6 +26,9 @@ web:
   src:
     frontend:
       path: /scrutiny/web
```

Comment:
Could it be a good idea to rename this on reder to disginguish `path` from `basepath`? Might be better to change it now than delay this for down the road.